src/main/java/seedu/address/model/game/DecreasingMode.java

```diff
@@ -0,0 +1,90 @@
+package seedu.address.model.game;
+
+import java.util.Date;
+
+import seedu.address.model.task.Task;
+
+// @@author chikchengyao
+
+/**
+ * Offers a mode where full XP is awarded for tasks done more than a certain time period
```

Comment:
it'll be clearer to say tasks done earlier than a certain time period, compared to more than a certain time period?